README.md

```diff
@@ -411,394 +335,7 @@ Start a server on port 8000, the API for the server is described in the [REST AP
-  command {
-    echo '${salutation}, ${name}!'
-  }
-  output {
-    String response = read_string(stdout())
-  }
-}
-
-workflow test {
-  String greeting
-  call hello {
-    input: salutation=greeting
-  }
-  call hello as hello2 {
-    input: salutation=greeting + " and nice to meet you"
-  }
-}
-```
-
-`String greeting` is referenced to satisfy the "salutation" parameter to both invocations of the `hello` task.
-
-The inputs required to run this would be:
-
-```
-{
-  "test.hello.name": "world",
-  "test.hello2.name": "boston",
-  "test.greeting": "hello"
-}
-```
-
-And this would produce the following outputs when run
-
-```
-$ java -jar cromwell.jar run hello.wdl hello.json
-... truncated ...
-{
-  "test.hello.response": "hello, world!",
-  "test.hello2.response": "hello and nice to meet you, boston!"
-}
-```
-
-## Using Files as Inputs
-
-So far every example has used the default type of `String` for every input.  Passing files along to tasks is simply a matter of defining the input type as `File`:
-
-```
-task grep {
-  File file
-
-  command {
-    grep -c '^...$' ${file}
-  }
-  output {
-    Int count = read_int(stdout())
-  }
-}
-
-workflow test {
-  call grep
-}
-```
-
-The `read_int()` function here would read the contents of its parameter, and interpret the first line as an integer and return that value as a WDL `Int` type.
-
-If I specified a file called `test_file` with the contents of:
-
-```
-foo
-bar
-baz
-quux
-```
-
-And then the inputs JSON file would be:
-
-```
-{
-  "test.grep.file": "test_file"
-}
-```
-
-The result of running this would be:
-
-```
-$ java -jar cromwell.jar run grep.wdl grep.json
-... truncated ...
-{
-  "test.grep.count": 3
-}
-```
-
-## Scatter/Gather
-
-Scatter blocks can be used to run the same call multiple times but only varying a specific parameter on each invocation.  Consider the following example:
-
-```
-task prepare {
-  command <<<
-    python -c "print('one\ntwo\nthree\nfour')"
-  >>>
-  output {
-    Array[String] array = read_lines(stdout())
-  }
-}
-
-task analysis {
-  String str
-  command <<<
-    python -c "print('_${str}_')"
-  >>>
-  output {
-    String out = read_string(stdout())
-  }
-}
-
-task gather {
-  Array[String] array
-  command <<<
-    echo ${sep=' ' array}
-  >>>
-  output {
-    String str = read_string(stdout())
-  }
-}
-
-workflow example {
-  call prepare
-  scatter (x in prepare.array) {
-    call analysis {input: str=x}
-  }
-  call gather {input: array=analysis.out}
-}
-```
-
-This example calls the `analysis` task once for each element in the array that the `prepare` task outputs.  The resulting outputs of this workflow would be:
-
-```
-{
-  "example.analysis.out": ["_one_", "_two_", "_three_", "_four_"],
-  "example.gather.str": "_one_ _two_ _three_ _four_",
-  "example.prepare.array": ["one", "two", "three", "four"]
-}
-```
+For many examples on how to use WDL see [the WDL site](https://github.com/broadinstitute/wdl/tree/develop#getting-started-with-wdl)
```

Comment:
For reviewers note that this is intended as a placeholder link. The eventual home will be somewhere on the official site once that's rolled out in a couple of weeks